src/components/Episodes/index.js

```diff
@@ -1,6 +1,8 @@
 import data from "./data.js";
+import ACTIONS from "./actions.js";
+import reducer from "./reducer.jsx";
 import RENDER from "./renderUtils.jsx";
 import { EpisodesContext, EpisodesProvider } from "./context.jsx";
 
-export { data, RENDER, EpisodesContext, EpisodesProvider };
+export { data, RENDER, reducer, ACTIONS, EpisodesContext, EpisodesProvider };
```

Comment:
not for now, but for another time, I'm thinking we should maybe just export `Context` and `Provider`, and then we can do:

```
import { Context as EpisodesContext } from "COMPONENTS/Episodes/";
```

Everything else is pretty neatly generic now, so it feels strange that this one's not! Super low priority, though, I might just do that on a branch myself at some point this weekend - I've already forced you to arbitrarily rename enough things 😅